t/lib/OpenQA/SeleniumTest.pm

```diff
@@ -58,30 +62,45 @@ sub start_app {
         }
 
         # Run openQA in test mode - it will mock Scheduler and Websockets DBus services
+        log_info("starting web UI\n");
         $ENV{MOJO_MODE} = 'test';
-        my $daemon = Mojo::Server::Daemon->new(listen => ["http://127.0.0.1:$mojoport"], silent => 1);
+        my $daemon = Mojo::Server::Daemon->new(
+            listen => ["http://127.0.0.1:$mojoport"],
+            silent => 1,
+        );
         $daemon->build_app('OpenQA::WebAPI');
-        $daemon->run;
+        $daemon->start(gru => 'run') if ($args->{with_gru});
```

Comment:
This looks like a typo, `Mojo::Server::Daemon` does not accept arguments.

Reply:
It is not a typo but a leftover from experimenting. Removed.